homeassistant/components/smsapi/notify.py

```diff
@@ -0,0 +1,50 @@
+"""Support for the SMSApi platform."""
+import logging
+
+from smsapi.client import SmsApiPlClient
+from smsapi.exception import SmsApiException
+import voluptuous as vol
+
+from homeassistant.components.notify import (
+    PLATFORM_SCHEMA, BaseNotificationService)
+from homeassistant.const import CONF_ACCESS_TOKEN, CONF_RECIPIENT
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_ACCESS_TOKEN): cv.string,
+    vol.Required(CONF_RECIPIENT): cv.string,
+})
+
+
+def get_service(hass, config, discovery_info=None):
+    """Get the SMSApi notification service."""
+
+    smsapi_client = SmsApiPlClient(access_token=config[CONF_ACCESS_TOKEN])
+
+    try:
+        smsapi_client.account.balance()
+        return SMSApiNotificationService(
```

Comment:
Please move this line out of the try... except block to the bottom of the function.

We only want the line that can error within try... except.